ui/lesson/TransactionsChallenge/index.tsx

```diff
@@ -233,16 +246,29 @@ const TransactionChallenge: FC<ITransactionProps> = ({
     }
   }
 
+  const handlePageViewChange = (view) => {
+    setActivePageView(view)
+  }
+
   return (
     <Lesson
       direction={
         isSmallScreen ? LessonDirection.Vertical : LessonDirection.Horizontal
       }
+      onViewChange={handlePageViewChange}
     >
+      <LessonTabs items={tabData} classes="px-4 py-2 w-full" stretch={true} />
       {children}
-      <div className="flex h-[calc(100vh-70px-48px)] w-full flex-shrink-0 flex-col justify-between border-white/25 md:h-[calc(100vh-70px)] md:max-w-[50vw] md:border-l">
+      <div
+        className={clsx(
+          'flex h-[calc(100vh-70px-48px)] w-full flex-shrink-0 flex-col justify-between border-white/25 md:h-[calc(100vh-70px)] md:max-w-[50vw] md:border-l',
+          {
+            'hidden md:flex': activePageView === LessonView.Info,
+          }
+        )}
+      >
         <div className="flex h-full flex-col gap-4 ">
-          <div className=" min-h-[65px] w-full border-b border-white/25 ">
+          <div className="min-h-[65px] w-full overflow-x-auto border-y border-white/25 md:border-t-0">
```

Comment:
A niit here would be if we can hide the scroll or change the default scroll on windows os

Reply:
you mean like just make sure it is scrolled to the end on render?